# **SOLUCIONES EJERCICIOS SECCION 4**  

<br></br>


## **Ejercicio 1**  

1, 3 y 4.

<br></br>


## **Ejercicio 2**  

```
de
```

<br></br>


## **Ejercicio 3**  

Cuál es el resultado esperado del siguiente código?
El código genera una excepción ```ValueError```.  

#  
<br></br>

- [Ejercicios](Sec4-ej.md)
<br></br>  

#  

[Volver a: Seccion 4 - Cadenas en acción](_Seccion4.md)  

[Volver a: Módulo 2 - Cadenas, métodos de listas y excepciones](../README.md)